docs/tf/link_pred.rst

```diff
@@ -120,6 +120,7 @@ Now we can initialize the graph engine in local mode:
     ...    storage_type=PartitionStorageType.memory,
     ...    config_path="",
     ...    stream=False,
+    ...    enable_threadpool=False,
```

Comment:
This defaults to false and can be removed right?

Reply:
We need to add it, otherwise there will be a test break: "enable_threadpool is not in Namespaces".